app/controllers/mixins/ems_common_angular.rb

```diff
@@ -431,6 +431,7 @@ def build_credentials(ems, mode)
          ems.supports_authentication?(:bearer) && !params[:default_password].blank?
         creds[:hawkular] = {:auth_key => params[:default_password], :userid => "_"}
         creds[:bearer] = {:auth_key => params[:default_password]}
+        creds.delete(:default)
```

Comment:
@Fryguy I like @AparnaKarve 's  idea in https://github.com/ManageIQ/manageiq/pull/9947#issuecomment-234079803, WDYT ?